### ISSUE TITLE
Make flutter inspector accessible

### ISSUE THREAD

**Issue by jacob314** [flutter/flutter#11678]
The flutter inspector does not currently support accessibility.

It isn't clear exactly what accessibility support for the inspector should look like. For example, if the semantics debugger is made accessible perhaps it should be used when using the inspector in accessibility mode.

**Comment by darshankawar** [comment]
Hi @jacob314 Can this issue be closed ?
Thanks.

**Comment by jacob314** [comment]
This issue should not be closed.
